### ISSUE TITLE
row class breaks layout for bootstrap form nested in modal

### ISSUE THREAD

**Issue by hanhanhan** [django-crispy-forms/django-crispy-forms#805]
* Package version: 1.7.2
* Bootstrap version 4.0.0 stable

The new "row" class being added from 1.7.0 -> 1.7.2:
[<div{% if formactions.attrs %} {{ formactions.flat_attrs|safe }}{% endif %} class="form-group row">](https://github.com/django-crispy-forms/django-crispy-forms/blob/71c75a21639191f52107cc0739518e02b9f670b2/crispy_forms/templates/bootstrap4/layout/formactions.html#L1)

Breaks layout when used with a bootstrap 4 form nested in a modal.

[What it looks like with django-crispy-forms 1.7.0](https://codepen.io/hanhanhan/pen/OvOeOW?editors=1011)
[What it looks like with django-crispy-forms 1.7.2](https://codepen.io/hanhanhan/pen/ZxadBo?editors=1011)

I believe the  `formactions` comes from adding a button at the bottom of the modal, like so -- only the layout on the buttons is shifted left:

```
            FormActions(
                Submit('stop', 'Stop', css_class='btn-primary'),
                Submit('go', 'Go', css_class='btn-danger')
            )
```

**Comment by zoidbergwill** [comment]
Whoops, looks like I missed `formactions.html` in https://github.com/zoidbergwill/django-crispy-forms/commit/2baf139e9c125650c996ee5cd5b2af868c7f8941#diff-0f9e2d08d3884dbef59475bf2552fa0e when I changed it to only add `row` to horizontal forms.

**Comment by serlachner** [comment]
Any chance this issue gets fixed? I got a related problem when using FormActions on a LoginForm. The added `row` class results in a margin-x of -15px which puts the button out of place according to the other layout elements.

Form:
```python
class LoginForm(AuthenticationForm):
    def __init__(self, *args, **kwargs):
        super(LoginForm, self).__init__(*args, **kwargs)

        self.helper = FormHelper()
        self.helper.layout = Layout(
            'username',
            'password',
            FormActions(
                Submit('login', 'Login'),
            ),
        )
```
Generated HTML (only FormActions):
```html
...
<div class="form-group row"> 
  <div class=""> 
    <input type="submit" name="login" value="Login" class="btn btn-primary" id="submit-id-login"/> 
  </div>
</div>
...
```
Expected HTML "row missing" (only FormActions):
```html
...
<div class="form-group"> 
  <div class=""> 
    <input type="submit" name="login" value="Login" class="btn btn-primary" id="submit-id-login"/> 
  </div>
</div>
...
```

**Comment by smithdc1** [comment]
Fixed by #919
